### PR TITLE
[FIX] web_editor: nonuniform behaviour toggle list maad

### PR DESCRIPTION
**Pull request by maad-odoo**
Before this commit:

Toggling the list with style like h1, h2, etc.. would break the list.

After this commit:

Toggling through the list with style does not break the list.

Task-id: 3269908

### CODE REVIEW

File: addons/web_editor/static/lib/odoo-editor/src/commands/commands.js [hunk 1]
```diff
@@ -528,7 +528,7 @@ export const editorCommands = {
                 const block = closestBlock(node);
                 if (!['OL', 'UL'].includes(block.tagName)) {
                     const ublock = block.closest('ol, ul');
-                    ublock && getListMode(ublock) == mode ? li.add(block) : blocks.add(block);
+                    ublock && getListMode(ublock) == mode ? li.add(block) : block.parentNode.tagName === 'LI' ? blocks.add(block.parentNode) : blocks.add(block);
```
Comment (dmo-odoo) [inline addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:531]: Looking at your fix, I feel like the issue is that the current code does not handle blocks that are inside LI correctly. Could you please try this and tell me if it works ? Thanks !
```js
let block = closestBlock(node);
if (!['OL', 'UL'].includes(block.tagName)) {
    block = block.closest('li') || block;
    const ublock = block.closest('ol, ul');
```

Comment (maad-odoo) [reply addons/web_editor/static/lib/odoo-editor/src/commands/commands.js:531]: Sure.

Comment (maad-odoo) [reply addons/web_editor/static/lib/odoo-editor/src/commands/commands.js]: @dmo-odoo I tried the changes, and it's working fine. I have updated the code, please check.

Comment (dmo-odoo) [reply addons/web_editor/static/lib/odoo-editor/src/commands/commands.js]: Awesome, thanks !

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**